build/index.ts

```diff
@@ -39,6 +39,7 @@ import {
   postProcessExternalLinks,
   postProcessSmallerHeadingIDs,
 } from "./utils.js";
+import webFeatures from "web-features/index.json" assert { type: "json" };
```

Comment:
🟢 Move import to the top so that the order is 1. node:* imports, 2. npm imports, 3. local imports